### ISSUE TITLE
LDAP Configuration for Jitsi

### ISSUE THREAD

**Issue by logicdefyer** [jitsi/docker-jitsi-meet#99]
Hi ,
I have been following this guide and modifying the files inside the proxidy container however it seems i am getting an internal error once its all done.

https://booting-rpi.blogspot.com/2015/09/using-ldap-authentication-with-jitsi.html

I have modified the prosody.cfg and jitsi .meet.cfg inside the prosody container config section

Any tips to get ldap working with jitsi , i already have ldap working with rocketchat/gitlab/nextcloud

**Comment by saghul** [comment]
See https://github.com/jitsi/docker-jitsi-meet/tree/dev#authentication-using-ldap

Note that that';s on the `dev` branch so you'll have to build the images yourself. The provided `Makefile` will help you with that.

**Comment by MaxNoe** [comment]
Is this still unreleased? Or are the dockerhub images no up to date?

**Comment by saghul** [comment]
It's already released.

**Comment by MaxNoe** [comment]
thx. I'm having trouble getting it to work with our ldap. It is using starttls with anonymous bind.

E.g. this works: `ldapsearch -x -Z -h ldap.ourdomain.org -b ou=people,dc=ourdomain,dc=de '(uid=maxnoe)'`, but the log says on login attempt:

```
prosody_1  | saslauthd[239] :released accept lock
prosody_1  | saslauthd[222] :acquired accept lock
prosody_1  | saslauthd[239] :attempting a read lock on slot: 747
prosody_1  | saslauthd[239] :[login=mnoethe] [service=xmpp] [realm=meet.jitsi]: not found, update pending
prosody_1  | saslauthd[239] :attempting to release lock on slot: 747
prosody_1  | saslauthd[239] :auth failure: [user=maxnoe] [service=xmpp] [realm=meet.jitsi] [mech=ldap] [reason=Unknown]
prosody_1  | saslauthd[239] :response: NO
```

**Comment by saghul** [comment]
TBH I'm not very familiar with the LDAP part since that was contrbiuted by @netaskd. Maybe they can help if they have some time.

**Comment by MaxNoe** [comment]
.env:

```
AUTH_TYPE=ldap
LDAP_URL=ldap://ldap.ourdomain.org
LDAP_BASE=ou=people,dc=ourdomain,org
# anonymous bind
#LDAP_BINDDN=
#LDAP_BINDPW=
LDAP_FILTER='(uid=%u)'
LDAP_AUTH_METHOD=bind
LDAP_VERSION=3
LDAP_USE_TLS=1
```

**Comment by MaxNoe** [comment]
Ah okay, there seem to options missing for starttls and some other stuff.

Let me see if I can open a PR tomorrow.

**Comment by MaxNoe** [comment]
Turns out anonymous binds were also not possible, fix in #206
